File: Swift-C++-Testsuite/test/collectionTypes/C++/Sets/Subset.cpp
#include <set>
#include <algorithm>

int main() {
        std::set<int> oddDigits = {1, 3, 5, 7, 9};  // Create a set of odd digits
        std::set<int> evenDigits = {0, 2, 4, 6, 8};  // Create a set of even digits

        bool isSubset = std::includes(evenDigits.begin(), evenDigits.end(), oddDigits.begin(), oddDigits.end());  // Check if oddDigits is a subset of evenDigits
    }

File: Swift-C++-Testsuite/test/collectionTypes/Swift/Sets/Subset.swift
@main
struct MyMain {

    static func main() {
        let oddDigits: Set = [1, 3, 5, 7, 9]
        let evenDigits: Set = [0, 2, 4, 6, 8]

        let isSubset = oddDigits.isSubset(of: evenDigits)
    }
}
